ripe/atlas/tools/commands/probes.py

```diff
@@ -166,6 +166,15 @@ def add_arguments(self):
                 "command."
             )
         )
+        self.parser.add_argument(
+            "--status",
+            type=int,
+            choices=[0, 1, 2, 3],
```

Comment:
yes i started doing that in the beginning but then i saw i had to do translation stuff to pass it to api (capital first letter and never connected has a space), so i said naah. also the multiple values is not supported on the API, so it would be doable on the cli without multiple queries.
So i think I will leave it as it is and someone can prettify it later when we add the multiple status feature on the api